### PR TITLE
Parse attributes of import expression with estree plugin

### LINKED ISSUE #13280: [Bug]: `@babel/eslint-parser` missing import assertion info in AST
### 💻

- [ ] Would you like to work on a fix?

### How are you using Babel?

@babel/eslint-parser

### Input code

```js
import("1.json", {type: "json"})
```

### Configuration file name

_No response_

### Configuration

N/A

### Current and expected behavior

```text
require('@babel/eslint-parser').parseForESLint('import("1.json", {type: "json"})', {requireConfigFile: false, babelOptions: {parserOpts:{plugins: ["importAssertions"]}}}).ast.body[0].expression
Node {
  type: 'ImportExpression',
  start: 0,
  end: 32,
  loc:
   SourceLocation {
     start: Position { line: 1, column: 0 },
     end: Position { line: 1, column: 32 },
     filename: undefined,
     identifierName: undefined },
  range: [ 0, 32 ],
  leadingComments: undefined,
  trailingComments: undefined,
  innerComments: undefined,
  extra: undefined,
  source:
   Node {
     type: 'Literal',
     start: 7,
     end: 15,
     loc:
      SourceLocation {
        start: [Position],
        end: [Position],
        filename: undefined,
        identifierName: undefined },
     range: [ 7, 15 ],
     leadingComments: undefined,
     trailingComments: undefined,
     innerComments: undefined,
     value: '1.json',
     raw: '"1.json"' } }
```

There is no information about the second argument.

I understand that ESTree hasn't added support for this yet, but maybe we can put in `extra` or something?

### Environment

I don't thinks this is useful

### Possible solution

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by sosukesuzuki**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            |  Fixes #13280
| Patch: Bug Fix?          | Y
| Tests Added + Pass?      | Yes
| License                  | MIT

<a href="https://gitpod.io/#https://github.com/babel/babel/pull/13284"></a>

### CODE REVIEW

**Review by JLHwung — APPROVED** [review]
Thanks!

File: packages/babel-parser/src/types.js [hunk 1]
```diff
@@ -1095,6 +1095,7 @@ export type EstreeMethodDefinition = NodeBase & {
 export type EstreeImportExpression = NodeBase & {
   type: "ImportExpression",
   source: Expression,
+  attributes: Expression | null,
```
Comment (JLHwung) [inline packages/babel-parser/src/types.js:1098]:
```suggestion
  attributes?: Expression | null,
```
The `attributes` is not assigned when the importAssertion plugin is not enabled.

**Comment by nicolo-ribaudo** [comment]
"Bug Fix" might be a bit of a stretch here, but I can convince myself that this isn't a new feature by saying that we were already parsing the import attributes but accidentally dropping them completely from the AST :shrug:

**Review by nicolo-ribaudo — APPROVED** [review]

**Comment by fisker** [comment]
Thanks for the quick fix.

**Pull request merged**